packages/block-library/src/quote/deprecated.js

```diff
@@ -109,7 +189,9 @@ const v2 = {
 			type: 'string',
 		},
 	},
-	migrate: migrateToQuoteV2,
+	migrate( attributes ) {
+		return migrateTextAlign( migrateToQuoteV2( attributes ) );
```

Comment:
As noted earlier, this also needs fixing to accept the array returned by `migrateToQuoteV2`